remote/remote.go

```diff
@@ -956,3 +958,7 @@ func isJSON(str string) bool {
 	var js json.RawMessage
 	return json.Unmarshal([]byte(str), &js) == nil
 }
+
+func isJSONObject(str string) bool {
+	return -1 != strings.Index(str, "{") && isJSON(str)
```

Comment:
Good catch. 
I have a change yet to be pushed that use the contains. HasPrefix is way better though.
Considering the JSON spec we should take into account the whitespace too. 
I'll think about it because with HasPrefix we risk to miss some valid JSON along the way. 

For example this JSON with extra spaces is valid.
 <pre>    {"test":1}    </pre>